src/request.rs

```diff
@@ -109,13 +109,11 @@ pub(crate) async fn request<Input: Serialize, Output: DeserializeOwned + 'static
 
     const CONTENT_TYPE: &str = "Content-Type";
     const JSON: &str = "application/json";
-    let user_agent = qualified_version();
 
     // The 2 following unwraps should not be able to fail
     let mut mut_url = url.clone().to_string();
     let headers = Headers::new().unwrap();
     headers.append("Authorization: Bearer", apikey).unwrap();
-    headers.append("User-Agent", &user_agent).unwrap();
```

Comment:
Can you change this to send this information by using the query param?

Reply:
Good idea